VARIABLES.md

```diff
@@ -996,6 +1044,14 @@ Default:  "{{ssl_enabled}}"
 
 ***
 
+### control_center_log_dir
+
+Only use to customize directory Control Center writes log files to. Default location is /var/log/confluent/control-center. NOTE- control_center.appender_log_path is deprecated
```

Comment:
Modify this variable to customize the directory that Control Center writes log files to. Default location is /var/log/confluent/control-center. NOTE- control_center.appender_log_path is deprecated.